### ISSUE TITLE
[0.9.3.0 beta staging-1954]RPCResponse

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#20949]
Happened randomly on tool switch/movement from building to digging

```
[07:36:29.903] Failed to handle RPCResponseSystem.NullReferenceException: Object reference not set to an instance of an object.
  at HasViewTooltipNoClick.OnGotBSON (Eco.Shared.Serialization.BSONObject info) [0x00000] in <00000000000000000000000000000000>:0 
  at Steamworks.Callback`1+DispatchDelegate[T].Invoke (T param) [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.Networking.RPCManager+<>c__DisplayClass22_0`1[T].<Call>b__0 (System.Object o) [0x00000] in <00000000000000000000000000000000>:0 
  at Steamworks.Callback`1+DispatchDelegate[T].Invoke (T param) [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.Networking.RPCManager.HandleQueryResponse (Eco.Shared.Serialization.BSONObject bson) [0x00000] in <00000000000000000000000000000000>:0 
  at NetworkManager.Eco.Shared.Networking.INetworkEventHandler.ReceiveEvent (Eco.Shared.Networking.INetClient client, Eco.Shared.Networking.NetworkEvent netEvent, Eco.Shared.Serialization.BSONObject bson) [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.Networking.NetObject.ReceiveEvent (Eco.Shared.Networking.INetClient client, Eco.Shared.Networking.NetworkEvent netEvent, Eco.Shared.Serialization.BSONObject bsonObj) [0x00000] in <00000000000000000000000000000000>:0 
  at ClientPacketQueueHandler.TryFetchNextClientUpdate (Eco.Shared.Time.TimeLimit timeLimit) [0x00000] in <00000000000000000000000000000000>:0 
  at ClientPacketHandler.HandleNetworkEvents (Eco.Shared.Time.TimeLimit timeLimit) [0x00000] in <00000000000000000000000000000000>:0 
  at System.Action`1[T].Invoke (T obj) [0x00000] in <00000000000000000000000000000000>:0 
  at FramePlanner.PlannerGroup.OnUpdate () [0x00000] in <00000000000000000000000000000000>:0 
  at FramePlanner.FramePlannerSystem.OnUpdate () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystemGroup.UpdateAllSystems () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.Application+LowMemoryCallback.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
```

**Comment by Daniel-Magulado** [comment]
Tested on 0.9.3 Staging-1971

Happens more frequently when building. Most especially when switching between fill types.

Adding log.
[RCP Error.txt](https://github.com/StrangeLoopGames/EcoIssues/files/6265634/RCP.Error.txt)

> [11:06:05.552] Disconnected with error: Failed to handle RPCResponse
EcoEngine.Logging.LogManager:LogFormat(LogType, Object, String, Object[])
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogWarning(Object)
NetworkManager:OnClientDisconnected(String)
DG.Tweening.Core.DOSetter`1:Invoke(T)
Eco.Shared.Networking.NetworkClient:HandleDisconnect(String)
Eco.Shared.Networking.Udp.UdpDisconnectedHandler:Invoke(Object, DisconnectReason)
Eco.Networking.ENet.ENetUdpClient:OnDisconnected(ENetConnection, DisconnectReason)
Unity.Serialization.<>c:.cctor()
Eco.Networking.ENet.ENetUdpPeer:Service(Int32)
<>c__DisplayClass65_0:<Initialize>b__0()
System.Threading.ThreadStart:Invoke()
System.Threading.ContextCallback:Invoke(Object)
System.Threading.ExecutionContext:RunInternal(ExecutionContext, ContextCallback, Object, Boolean)
System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object)
System.Threading.ThreadStart:Invoke()

**Comment by SlayksWood** [comment]
#21385

**Comment by SlayksWood** [comment]
merged to #21486
